src/admin/components/utilities/Preferences/index.tsx

```diff
@@ -44,7 +44,6 @@ export const PreferencesProvider: React.FC = ({ children }) => {
   }, [api, preferencesRef, serverURL]);
 
   const setPreference = useCallback(async (key: string, value: unknown): Promise<void> => {
-    if (preferencesRef.current[key] && JSON.stringify(await preferencesRef.current[key]) === JSON.stringify(value)) return;
```

Comment:
@DanRibbens I had to remove this line to ensure that we were always getting an up-to-date version of the preferences bound to context.